content/en/dora_metrics/deployments/apm.md

```diff
@@ -0,0 +1,129 @@
+
+### Synchronize repository metadata to Datadog
+
+<!--
+The Following tabs were mostly copied from the Source Code Integration docs until we find a way to document this in a shared page
+https://docs.datadoghq.com/integrations/guide/source-code-integration/?tab=github#synchronize-your-repository-metadata
+-->
+
+<div class="alert alert-warning">
+GitHub workflows running on <a href="https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request"> <code>pull_request</code> trigger </a> are not currently supported by the GitHub integration.
+If you are using the <code>pull_request</code> trigger, use the alternative method.
+</div>
+
+If the [GitHub integration][1] is not already installed, install it on the [GitHub integration tile][2].
+
+When configuring the GitHub application:
+1. Select at least **Read** repository permissions for **Contents** and **Pull Requests**.
+2. Subscribe at least to **Push**, **PullRequest** and **PullRequestReview** events.
+
+To confirm that the setup is valid, select your GitHub application in the [GitHub integration tile][2] and verify that, under the **Features** tab, the **DORA Metrics: Collect Change Lead Time metric** feature is enabled.
+
+[1]: https://docs.datadoghq.com/integrations/github/
+[2]: https://app.datadoghq.com/integrations/github/
+
+### Handling multiple services in the same repository
+
+If the source code of multiple services is present in the same repository, further actions are needed to ensure that the Change Lead Time is calculated by taking into account only the commits affecting the specific service being deployed.
+To filter the commits measured to only the ones that affect the service, specify the source code glob file path patterns in the [service definition][5].
+
+If the service definition contains a **full** GitHub URL to the application folder, a single path pattern is automatically used.
+
+**Example (schema version v2.2):**
+
+```yaml
+links:
+  - name: shopist
+    type: repo
+    provider: github
+    url: https://github.com/organization/example-repository/tree/main/src/apps/shopist
+```
+
+DORA Metrics for the `shopist` service only consider the Git commits that include changes within `src/apps/shopist/**`. You can configure more granular control of the filtering with `extensions[datadoghq.com/dora-metrics]`.
+
+**Example (schema version v2.2):** 
+
+```yaml
+extensions:
+  datadoghq.com/dora-metrics:
+    source_patterns:
+      - src/apps/shopist/**
+      - src/libs/utils/**
+```
+
+DORA Metrics for the service `shopist` only consider the Git commits that include changes within `src/apps/shopist/**` or `src/libs/utils/**`.
+
+### Limitations
+
+- Change Lead Time for deployments identified via APM Deployment Tracking is only available for GitHub repositories.
```

Comment:
I haven't tested with other providers, but I think it should work, am I missing something? Users can send git metadata using `datadog-ci git-metadata upload`, and tag their telemetry with the repository url and commit sha.